index.d.ts

```diff
@@ -332,8 +332,8 @@ export interface Bot extends TypedEmitter<BotEvents> {
 
   craft: (
     recipe: Recipe,
-    count: number | null,
-    craftingTable: Block,
+    count?: number,
```

Comment:
Let's undo these changes

Reply:
Even craftingTable? It is indeed optional ^^

Reply:
Ok yeah i guess that's fine to keep then